Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -0,0 +1,34 @@
+require "language/go"
+
+class DockerMachineDriverXhyve < Formula
+  desc "Docker Machine driver for xhyve"
+  homepage "https://github.com/zchee/docker-machine-driver-xhyve"
+  url "https://github.com/zchee/docker-machine-driver-xhyve/archive/v0.2.1.tar.gz"
+  sha256 "11ed10c4ea45249e06c9efad0abf1c91b0ec9c4c2a0818ade78f19f41a17794c"
+
+  head "https://github.com/zchee/docker-machine-driver-xhyve.git"
+
+  depends_on :macos => :yosemite
+  depends_on "go" => :build
+  depends_on "docker-machine"
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/zchee/docker-machine-driver-xhyve").install contents
+
+    ENV["GOPATH"] = gopath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
+
+    cd gopath/"src/github.com/zchee/docker-machine-driver-xhyve" do
+      if build.head?
+        git_commithash = `git rev-parse --short HEAD 2>/dev/null`.delete!("\n")
+      end
+      system "go", "build", "-o", bin/"docker-machine-driver-xhyve", "-ldflags", "'-w -s'", "-ldflags", "-X 'github.com/zchee/docker-machine-driver-xhyve/xhyve.GitCommit=#{git_commithash}+Homebrew'", "./main.go"
```

Comment:
@mikemcquaid Case of install with `--HEAD` flag,

```bash
(xhyve-test) Calling .GetMachineName
(xhyve-test) Calling .DriverName
(xhyve-test) Calling .GetCreateFlags
(xhyve-test) Calling .SetConfigFromFlags
Running pre-create checks...
(xhyve-test) Calling .PreCreateCheck

# This line
(xhyve-test) DBG | ===== Docker Machine xhyve Driver Version 0.2.1 (896d3f5+Homebrew) =====

(xhyve-test) DBG |
(xhyve-test) DBG | executing: /usr/local/bin/VBoxManage -v
(xhyve-test) DBG | STDOUT: 5.0.12r104815
(xhyve-test) DBG |
(xhyve-test) DBG | STDERR:
(xhyve-test) Calling .GetConfigRaw
Creating machine...
(xhyve-test) Calling .Create
(xhyve-test) DBG | local Boot2Docker ISO version:  v1.9.1
(xhyve-test) Copying /Users/zchee/.docker/machine/cache/boot2docker.iso to /Users/zchee/.docker/machine/machines/xhyve-test/boot2docker.iso...
(xhyve-test) Creating VM...
(xhyve-test) Extracting vmlinuz64 and initrd.img from boot2docker.iso...
(xhyve-test) DBG | Mounting boot2docker.iso
```

If without `--HEAD` flag,

```bash
(xhyve-test) Calling .GetMachineName
(xhyve-test) Calling .DriverName
(xhyve-test) Calling .GetCreateFlags
(xhyve-test) Calling .SetConfigFromFlags
Running pre-create checks...
(xhyve-test) Calling .PreCreateCheck

# This line
(xhyve-test) DBG | ===== Docker Machine xhyve Driver Version 0.2.1 (+Homebrew) =====

(xhyve-test) DBG |
(xhyve-test) DBG | executing: /usr/local/bin/VBoxManage -v
(xhyve-test) DBG | STDOUT: 5.0.12r104815
(xhyve-test) DBG |
(xhyve-test) DBG | STDERR:
(xhyve-test) Calling .GetConfigRaw
Creating machine...
(xhyve-test) Calling .Create
(xhyve-test) DBG | local Boot2Docker ISO version:  v1.9.1
(xhyve-test) Copying /Users/zchee/.docker/machine/cache/boot2docker.iso to /Users/zchee/.docker/machine/machines/xhyve-test/boot2docker.iso...
(xhyve-test) Creating VM...
(xhyve-test) Extracting vmlinuz64 and initrd.img from boot2docker.iso...
(xhyve-test) DBG | Mounting boot2docker.iso
```